Source/RunActivity/Viewer3D/Popups/HUDWindow.cs

```diff
@@ -583,6 +584,11 @@ void TextPageConsistInfo(TableData table)
             }
         }
 
+        /// <summary>
+        /// Calculates Whyte noatation for the vehicle
+        /// For duplex steam locomotives wheel axles can be grouped under the main object shape, and hence the multiple engines will be counted as a single grouping.
+        /// For multiple engines, the number of wheels defined in the attached axles will be used to determine the axles in the "non-bogie" groupings.
+        /// </summary>
         static string GetCarWhyteLikeNotation(TrainCar car)
```

Comment:
Hi Peter,

Instead of a lengthy method, why not split it into 2 with names that explain what is going on? 
Something like this:

```
        static string GetCarWhyteLikeNotation(TrainCar car)
        {
            if (car.WheelAxles.Count == 0)
                return "";

            var whyte = new List<string>();          
            var steamloco = car as MSTSSteamLocomotive;

            if (car is MSTSSteamLocomotive && steamloco.SteamEngines.Count > 1)
            {
                BuildMultiEngineNotation(car, whyte, steamloco.SteamEngines);
            }
            else // default axle computation - used for most wheel configurations
            {
                BuildSingleEngineNotation(car, whyte);
            }
            return String.Join("-", whyte.ToArray());
        }
```

Reply:
What is the value of creating two "sub" methods? Each of the methods is only processed once, so there wouldn't be a great saving(?).

Reply:
Hi Peter,

No saving in performance, but better for readability. Partly because the nesting of the if ,, then .. else is less deep, but also because the method names declare their purpose and the need for comments is reduced.

However readability is subjective and this is just a suggestion.

Chris